other/version-sync

```diff
@@ -26,3 +26,7 @@ update 'configure.ac' 's/AC_INIT(\[tox\], \[.*\])/AC_INIT([tox], ['$VER'])/'
 update 'toxcore/tox.api.h' 's/\(const VERSION_MAJOR *= \).*;/\1'$MAJOR';/'
 update 'toxcore/tox.api.h' 's/\(const VERSION_MINOR *= \).*;/\1'$MINOR';/'
 update 'toxcore/tox.api.h' 's/\(const VERSION_PATCH *= \).*;/\1'$PATCH';/'
+
+update 'CMakeLists.txt' 's/^\(set(PROJECT_VERSION_MAJOR \"\).*")/\1'$MAJOR'")/'
+update 'CMakeLists.txt' 's/^\(set(PROJECT_VERSION_MINOR \"\).*")/\1'$MINOR'")/'
+update 'CMakeLists.txt' 's/^\(set(PROJECT_VERSION_PATCH \"\).*")/\1'$PATCH'")/'
```

Comment:
Uh, you are not supposed to call this script manually. It's supposed to be called by cmake. When releasing a new version, you update the version number in the CMakeLists.txt and run cmake, which will update the version everywhere. At least that'she how I understand it's supposed to be done.